extensions/wikia/Chat2/js/controllers/controllers.js

```diff
@@ -165,6 +165,7 @@ var NodeRoomController = $.createClass(Observable, {
 		this.socket.bind('longMessage', $.proxy(this.onLongMessage, this));
 
 		this.socket.bind('logout', $.proxy(this.onLogout, this));
+		this.socket.bind('freeze', this.onFreeze.bind(this));
```

Comment:
Are we sure that `Function.prototype.bind` is present in all browsers that we support? Let's be consistent with the rest of the code here and use jQuery's `bind`.